Heffner_Daniel/Test_Driven_Development/RSpecRoutes/spec/controllers/dojos_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+RSpec.describe DojosController, type: :controller do
+
+  describe "routing" do
+    it 'routes to #index' do
+      expect(:get => "/").to route_to('dojos#index')
+    end
+    it 'routes to #world' do
+      expect(:post => "/hello").to route_to(:controller => "dojos", :action => "world")
+    end
+    it 'routes to #ninjas' do
+      expect(:get => "/ninjas").to route_to('dojos#ninjas')
```

Comment:
I like being consistent in my tests, choose 'dojos#ninjas' or (:controller => "dojos", :action => "world") and use them in all your tests. It helps set convention.